docs/Documentation/openvino_ecosystem.md

```diff
@@ -104,5 +105,18 @@ More resources:
 * `PyPI <https://pypi.org/project/datumaro/>`__  
 * `GitHub <https://github.com/openvinotoolkit/datumaro>`__  
 
+Compile Tool 
+#############
+
+Compile tool is deprecated since the 2023.0 OpenVINO release. If you need to compile a model for inference on a specific device, use the following script: 
+
+.. code-block:: python
```

Comment:
I propose to add code snippets and reuse it here: https://github.com/openvinotoolkit/openvino/tree/master/docs/snippets

Also please add the snippet for C++

Reply:
> I propose to add code snippets and reuse it here: https://github.com/openvinotoolkit/openvino/tree/master/docs/snippets
> 
> Also please add the snippet for C++

snippets added but I need help with the cpp content